### DISCUSSION TITLE
spawner problem

### DISCUSSION THREAD

**Discussion by CRAFTINGHARD1234** [BG-Software-LLC/SuperiorSkyblock2#380] [Denied Feature Requests] [answered]
mobs are not spawning from spawner in my island 
i check the everything
from island upgrade , everything 
"Nerfing mobs spawned from spawners: false"
pls can anyone one tell how to solve the problem

**Comment by OmerBenGera** [comment, marked as answer]
Do not open discussions about issues you experience. If you believe there is a bug, then open an issue instead.
